README.md

```diff
@@ -187,6 +187,18 @@ To control the number of scenes seeded per product for `s4e-sync-1` dataset use
 To load an unlimited number set its value to -1.
 
 
+#### Scene updates from AMQP queue
+
+Application can be configured to listen to notifications from an AMQP queue with S3 object updates.
+
+In profile `development` this feature is enabled, and in `production` it has to be enabled by setting
+`amqp.enabled=true` or `AMQP_ENABLED=true`.
+
```

Comment:
i would add that you can configure queues names by properties.

Reply:
Will do